wgsl/index.bs

```diff
@@ -7913,56 +7913,64 @@ variables=] for each root identifier.
 
 Two root identifiers <dfn noexport>alias</dfn> when they have the same
 [=originating variable=].
-It is a [=dynamic error=] if:
-* more than one aliased root identifiers access the same [=memory locations=], and
-* at least one access is a [=write access|write=], and
-* both accesses occur during the execution of the same call of the function
-
-Note: This aliasing restriction applies to memory locations written by
-[=function calls=] made in the function.
-
-Note: [=Originating variables=] cannot have aliased [=memory locations=].
-See [[#var-decls]] and [[#resource-interface]].
-
-<div class='example wgsl' heading='Aliased memory views'>
+A WGSL program [=shader-creation error|must not=] contain any potential aliases.
```

Comment:
It's not "any potential aliases".     They have to be conflicting: one a write, and the other a read or write.
And it must somehow be scoped for the execution lifetime of a single function.

Suggest something like this:

> Execution of a WGSL function [=shader-creation error|must not=] potentially access memory through aliased root identifiers, where one access is a write and the other is a read or a write.   This is determined by analyzing the program from the leaves ....